fmc/src/hand_off.rs

```diff
@@ -155,6 +155,33 @@ impl HandOff {
         }
     }
 
+    /// Retrieve runtime TCI (digest)
+    pub fn rt_svn(&self, env: &FmcEnv) -> u32 {
+        let ds: DataStore = self.fht.rt_svn_dv_hdl.try_into().unwrap_or_else(|_| {
+            caliptra_common::report_handoff_error_and_halt(
+                "Invalid RT SVN handle",
+                caliptra_error::CaliptraError::FMC_HANDOFF_INVALID_PARAM.into(),
+            )
+        });
+
+        // The data store is either a warm reset entry or a cold reset entry.
+        match ds {
+            DataVaultNonSticky4(dv_entry) => env.data_vault.read_warm_reset_entry4(dv_entry),
+            DataVaultSticky4(dv_entry) => env.data_vault.read_cold_reset_entry4(dv_entry),
+            _ => {
+                crate::report_error(
+                    caliptra_error::CaliptraError::FMC_HANDOFF_INVALID_PARAM.into(),
+                );
+            }
+        }
+    }
+
+    pub fn set_rt_dice_signature(&self, _sig: &Ecc384Signature) {
+        // TODO - implement
+        // Where should this be stored?
+        // No DV slots avaiable anymore.
```

Comment:
As discusses elsewhere, can store this directly in the FHT

Reply:
Agreed. We need a separate PR for that.

Reply:
Sounds good